.github/workflows/check-no-suggests.yaml

```diff
@@ -0,0 +1,66 @@
+# Workflow derived from https://github.com/r-lib/actions/tree/v2/examples
+# Need help debugging build failures? Start at https://github.com/r-lib/actions#where-to-find-help
+#
+# NOTE: This workflow only directly installs "hard" dependencies, i.e. Depends,
+# Imports, and LinkingTo dependencies. Notably, Suggests dependencies are never
+# installed, with the exception of testthat, knitr, and rmarkdown. The cache is
+# never used to avoid accidentally restoring a cache containing a suggested
+# dependency.
+#
+# Customization for `{lintr}`
+#
+# The following soft dependencies are not optional for testing:
+# `{patrick}`, `{rex}`, `{withr}`
```

Comment:
Why is rex a soft dep? We use it in our package code, no?

Reply:
shall we delete rex from the loadNamespace calls? I believe it was added by mistake

Reply:
Yeah, I think it was introduced accidentally. Made a separate PR to fix that.